lib/scientist/experiment.rb

```diff
@@ -121,6 +121,16 @@ def compare(*args, &block)
     @_scientist_comparator = block
   end
 
+  # A block which compares two experimental errors.
+  #
+  # The block must take two arguments, the control Error and a candidate Error,
+  # and return true or false.
+  #
+  # Returns the block.
+  def error_compare(*args, &block)
```

Comment:
I think `compare_errors` would be a better name for the method, at least it reads better to me:

```ruby
science do |e|
  e.compare { |a, b| ... }
  e.compare_errors { |a, b| ... }
end
```

The instance variable doesn't need to change, though.